tests/zpill.py

```diff
@@ -105,9 +106,46 @@ def serialize(obj):
     raise TypeError("Type not serializable")
 
 
+# We cannot serialize datetime by default in json dumps
+# use this method to override default behavior
+def datetime_converter(obj):
+    if isinstance(obj, datetime):
+        return obj.isoformat()
+
+
+def save_response(self, service, operation, response_data,
+                  http_response=200):
+    """
+    Overrides the standard behavior for this function
+    so that it does not store response metadata and
+    account_ids are properly sanitized
+    """
+    pill.LOG.debug('save_response: %s.%s', service, operation)
+    filepath = self.get_new_file_path(service, operation)
+    pill.LOG.debug('save_response: path=%s', filepath)
+
+    # remove response metadata
+    if 'ResponseMetadata' in response_data:
+        response_data['ResponseMetadata'] = {}
+
+    # When there is owner info, set the name to a default
+    if 'Owner' in response_data and response_data['Owner']["DisplayName"] is not None:
```

Comment:
this is pretty specific to a single api call, we need to string replace account id on the output, note this conflicts with being able to call super above.

Reply:
Thanks Kapil, yeah I threw it in there because I noticed it on some of the json docs. I think I am overcooking the bird here :D